bundle/manifests/security-profiles-operator.clusterserviceversion.yaml

```diff
@@ -969,6 +969,8 @@ spec:
   links:
   - name: Security Profiles Operator
     url: https://github.com/kubernetes-sigs/security-profiles-operator
+  - name: Security Profiles Operator documentation
+    url: https://docs.openshift.com/container-platform/latest/security/security_profiles_operator/spo-understanding.html
```

Comment:
Maybe better to use overview page instead?
https://docs.openshift.com/container-platform/latest/security/security_profiles_operator/spo-overview.html